src/style.css

```diff
@@ -0,0 +1,109 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+  margin: 0;
+}
+
+.product-card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 29px 14px 14px;
+}
+
+.product-card__img {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: 0 auto;
+}
+
+.product-card__title {
+  font-size: 12px;
+  line-height: 15px;
+  font-weight: 500;
+  margin-top: 45px;
+}
+
+.product-card__code {
+  font-size: 11px;
+  font-weight: 400;
+  margin-top: 6px;
+}
+
+.product-card__rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 15px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 15px;
+  height: 15px;
+  margin-right: 4px;
+  background: center no-repeat url(images/star-active.svg);
+}
+
+.stars__star:last-child {
+  background: center no-repeat url(images/star.svg);
+}
+
+.revives {
+  font-size: 11px;
+  font-weight: 400;
+  text-align: right;
+  padding-bottom: 0;
+}
+
+.product-card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: baseline;
+  margin-top: 20px;
+}
+
+.product-card__prise-name {
+  font-size: 11px;
```

Comment:
You use `font-size: 11px;` three times in your code. Let's move it to the `body` as a common font